spec/index.bs

```diff
@@ -2769,6 +2839,21 @@ enum GPUFeatureName {
 };
 </script>
 
+<h4 id=gpubehaviorname data-dfn-type=enum>`GPUBehaviorName`
+<span id=enumdef-gpubehaviorname></span>
+</h5>
+
+Each {{GPUBehaviorName}} identifies a set of functionality which, if available,
+allows additional usages of WebGPU that may otherwise be invalid.
+A [=behavior=] is available of the user agent supports it.
+An application does not request [=behaviors=].
+
+<script type=idl>
+enum GPUBehaviorName {
+  "wgsl"
```

Comment:
Web IDL disallows empty enums.   I added "wgsl" as a placeholder, but it feels awkward.

Reply:
It may be that we should not create the behaviors concept until we actually have the first use case. Then we wouldn't need to create a placeholder like this.

Reply:
The enum GPUBehaviorName is not (and will not be) used anywhere in the API IDL so it doesn't need to be written in IDL. The list of strings, currently empty, can be written in prose.